lib/perfdata/influxdbcommonwriter.cpp

```diff
@@ -109,20 +109,13 @@ void InfluxdbCommonWriter::Pause()
 	Log(LogDebug, GetReflectionType()->GetName())
 		<< "Flushing pending data buffers.";
 
-	Flush();
-
-	/* Work on the missing tasks. TODO: Find a way to cache them on disk. */
-	Log(LogDebug, GetReflectionType()->GetName())
-		<< "Joining existing WQ tasks.";
```

Comment:
Why? What’s the problem?

Reply:
What do you mean? Why this message was removed? Because it would provide no meaningful information anymore compared to the one before. The wording could be integrated into the other message though.